modules/aws/sonar-base-instance/iam_role.tf

```diff
@@ -1,9 +1,9 @@
 #################################
-# Hub IAM role
+# DSF node IAM role
 #################################
 
 locals {
-  role_arn  = var.role_arn != null ? var.role_arn : try(aws_iam_role.dsf_hub_role[0].arn, null)
+  role_arn  = var.role_arn != null ? var.role_arn : try(aws_iam_role.dsf_node_role[0].arn, null)
```

Comment:
var.role_arn should be passed from hub's and gw's `main.tf` files

Reply:
Added (you don't miss anything do you?)